changelog/add-profile-links-filter

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fixed
+
+Add `sensei_results_completed_links` filter
```

Comment:
I think we can skip the changelog entry for this PR because we don't have a `development` changelog section yet ([it's in the works](https://github.com/Automattic/sensei/pull/6804)). For PRs with no changelog, you can use the `No Changelog` PR label to make CI happy.

Reply:
I don't think we should skip notifying people about changes like this just because we don't have it automated. It can be included with the rest of the changes in the change log.